frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -0,0 +1,126 @@
+<script setup lang="ts">
+import { ref, type PropType } from 'vue';
+import { FilterMatchMode } from 'primevue/api';
+import Column from 'primevue/column';
+import DataTable from 'primevue/datatable';
+
+import { routeItems } from '@/router/routeItem';
+import DataTableHeader from '@/components/managePermissions/table/DataTableHeader.vue';
+import type { FamAccessControlPrivilegeGetResponse } from 'fam-admin-mgmt-api/model';
+
+const props = defineProps({
+    loading: {
+        type: Boolean,
+        default: false,
+    },
+    delegatedAdmins: {
+        type: [Array] as PropType<FamAccessControlPrivilegeGetResponse[] | undefined>,
+        required: true,
+    },
+});
+
+const delegatedAdminFilters = ref({
+    global: { value: '', matchMode: FilterMatchMode.CONTAINS },
+    'user.user_name': { value: null, matchMode: FilterMatchMode.STARTS_WITH },
+    'user.user_type.description': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.role_name': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.parent_role.role_name': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+});
+
+const delegatedAdminSearchChange = (newvalue: string) => {
+    delegatedAdminFilters.value.global.value = newvalue;
+};
+</script>
+
+<template>
+    <!-- Hidden until functionality is available
+    <ConfirmDialog group="deleteAdmin">
+        <template #message>
+            <ConfirmDialogtext
+                :userName=""
+                :role=""
+            />
+        </template>
+    </ConfirmDialog> -->
+    <div class="data-table-container">
+        <div class="custom-data-table">
+            <DataTableHeader
+                btnLabel="Create delegated admin"
+                :btnRoute="routeItems.grantAppAdmin.path"
```

Comment:
this route later need to change as well, maybe let's put "#" for now

Reply:
sure, will do!